spec/dashboard-menu-view.spec.js

```diff
@@ -1,8 +1,8 @@
 var moment = require('moment');
 var Backbone = require('backbone');
-var DashboardInfoView = require('../src/dashboard-info-view');
+var DashboardMenuView = require('../src/dashboard-menu-view');
 
-describe('../src/dashboard-info-view', function () {
+describe('../src/dashboard-menu-view', function () {
```

Comment:
@acanimal and I thought it was clearer, since it's more of (top-level) menu/bar of some sort.
That said, I don't have a strong opinion, we could change it back if it's important (or come up with some better name  to represent this). What do you think?